static/setup.txt

```diff
@@ -624,6 +624,11 @@ plugin.tx_cfcleaguefe_competition {
         if.value = 1
         if.equals.field = markClub
       }
+      markClubsIsRunningGame {

```

Comment:
Allgemein macht es schon sinn, das zu trennen. In meinem Fall benötige ich aber auch beides.